### ISSUE TITLE
Fetch() on page request

### ISSUE THREAD

**Issue by iet-admin** [websanova/vue-auth#280]
Hi 

After a bit of help, can't make sense of something from the code or docs.

On the initialise options you can set:
`fetchData: {url: 'api/user/fetch', method: 'GET', enabled: true}`

When the 'enabled' flag is set to true, this seems to trigger a fetch() call on each complete page reload if the user is authenticated if I understand correctly. 

I've configured an API endpoint to handle this and return a user object in the response. My issue is that I don't know how to access this user object in the response? As I'm not manually calling fetch() I can't hook in to the success with the response item as far as I can see.

Basically I just want to reset the user data (as $auth.user(userobject)) does on a full page reload.

I guess the alternative is to either manually call $auth.fetch() in the base component to fire on each page reload and capture the user object from the success response but seems unnecessary if it's already being fired by the plugin.

Hope you can help.

**Comment by websanova** [comment]
Not really sure what you're asking.

You don't see anything when you do `console.log($auth.user());` ?

Also are you properly using `$auth.ready()` to make sure everything is fully loaded before trying to handle the user?

**Comment by iet-admin** [comment]
Yeah that's pretty much it, after full page reload. But just solved it by doing some debug in auth.js.

I was returning the user object as 'user' in the response from my fetch method on the server. Changed this to 'data' and seems to be working as expected now!

(Sorry if I wasn't clear in original post, was hard to explain!)

**Comment by iet-admin** [comment]
Or i could have overridden it like this I guess:

Vue.use(require('vue-auth'), {
    parseUserData: function (data) {
        return data.user;
    }
});

(It was in the docs, was just my lack of understanding!)
